docs/tutorials/configure/build-a-mock-robot.md

```diff
@@ -13,106 +13,120 @@ aliases:
 tags: ["mock", "simulation"]
 ---
 
-## Introduction
+This tutorial will show you how to build a mock robot using just your personal laptop so you can:
 
-In this post, we will show you how to build a mock robot using just your personal laptop so you can try using Viam without any robotic hardware.
-This is a great way to learn how to build robots [the Viam way](/viam).
+  - learn how to [configure](../../../manage/configuration/) robots with Viam and
+ - try using [Viam](/viam) without any robotic hardware.
 
-Most Viam components come with a _fake_ model that can be useful when testing.
-These fake components interact with Viam like real hardware, but of course, do not actually exist.
-We will be using these fake components to build out a mock robot and explore how to use Viam.
+Most Viam [components](../../../components/) have a _fake_ model that you can use for testing.
+These fake components interact with Viam like real hardware but do not actually exist.
```

Comment:
here I think it's worth giving more info about finding API call specifications (in github and on model pages) & the limitations of these right now-- I think I've seen some discussion about engineering is unsure about supporting full component functionality for fake models, and have seen some of them are more advanced with configuration & capabilities for testing API methods than others (for example, fake `arm` vs a fake `base`) 

At the least you could add in a sentence or two here directing to the fake component pages, even though they are mentioned below!

Reply:
Hmmm interesting. I'm hesitant to add any words about the extent to which different methods are or aren't supported if eng is still unsure about it such that it is inconsistent and may be in flux. 

RE directing them to fake component pages, I am hopeful that with the added link to components above, people who are very curious will click through and see them. If not, as you say they're linked below. I don't think adding links here will help since that's just config info, and we're about to tell them how to configure things so they should just keep reading and they'll get the info when they need it, in context.